src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/DirectiveAttributeTransitionCompletionItemProviderTest.cs

```diff
@@ -143,19 +144,6 @@ public void GetCompletionItems_AttributeAreaInNonComponentFile_ReturnsEmptyList(
         Assert.Empty(result);
     }
 
-    [Fact]
-    public void GetCompletionItems_OutsideOfFile_ReturnsEmptyList()
```

Comment:
I removed all of these types of tests, where it was really just testing when you call `LocateOwner` with an out-of-bounds range.